File: 2455-average-value-of-even-numbers-that-are-divisible-by-three/2455-average-value-of-even-numbers-that-are-divisible-by-three.cpp
class Solution {
public:
    int averageValue(vector<int>& nums) {
        vector<int>res;
        for(int i=0; i<nums.size();i++){
            if(nums[i]%6==0)
                res.push_back(nums[i]);
        }
         if(res.size()>0) return accumulate(res.begin(), res.end(),0)/res.size();
         return 0;

    }
       
};

File: 2455-average-value-of-even-numbers-that-are-divisible-by-three/2455-average-value-of-even-numbers-that-are-divisible-by-three.py
class Solution:
    def averageValue(self, nums: List[int]) -> int:
        cntr = 0
        evenSum = 0
        for i in nums:
            if i % 6 == 0:
                evenSum += i
                cntr += 1
        if cntr: return evenSum // cntr
        else: return 0